data/medibase.json

```diff
@@ -1144305,4 +1144305,4 @@
   "cims_class_link": "/india/drug/search?q=drugs+for+erectile+dysfunction&mtype=brand&code=10d",
   "cims_class": "Drugs for Erectile Dysfunction",
   "atc_classification": "G04BE03 - sildenafil ; Belongs to the class of drugs used in erectile dysfunction."
-}]
\ No newline at end of file
+}]
```

Comment:
run `git restore --source=master -- data/medibase.json && git commit -anm "undo changes"` to revert this change

Reply:
Thanks done. Updated the other requested changes as well.